src/test/kotlin/com/petqua/presentation/product/ProductReviewControllerSteps.kt

```diff
@@ -0,0 +1,34 @@
+import com.petqua.domain.product.review.ProductReviewSorter
+import com.petqua.domain.product.review.ProductReviewSorter.REVIEW_DATE_DESC
+import io.restassured.module.kotlin.extensions.Extract
+import io.restassured.module.kotlin.extensions.Given
+import io.restassured.module.kotlin.extensions.Then
+import io.restassured.module.kotlin.extensions.When
+import io.restassured.response.Response
+
+
+fun requestReadAllReviewProducts(
+    productId: Long,
+    sorter: ProductReviewSorter = REVIEW_DATE_DESC,
+    lastViewedId: Long = -1,
+    limit: Int = 20,
+    score: Int? = null,
+    photoOnly: Boolean = false
+): Response {
+    return Given {
+        log().all()
+        params(
+            "sorter", sorter.name,
+            "lastViewedId", lastViewedId,
+            "limit", limit,
+            "score", score,
+            "photoOnly", photoOnly
+        )
+    } When {
+        get("/products/{productId}/reviews", productId)
+    } Then {
+        log().all()
+    } Extract {
+        response()
+    }
```

Comment:
path variable 에 대한 값을 Given 절에서 세팅하면 어떨까요? 개인적으로 When 에서는 주소 외에 내용이 안 보일 때가 많더라구요

```suggestion
    return Given {
        log().all()
        params(
            "sorter", sorter.name,
            "lastViewedId", lastViewedId,
            "limit", limit,
            "score", score,
            "photoOnly", photoOnly
        )
        pathParam("productId", productId)
    } When {
        get("/products/{productId}/reviews")
    } Then {
        log().all()
    } Extract {
        response()
    }
```

Reply:
옹 저는 `get("/products/{productId}/reviews", productId)` 요게 직접적으로 보여서 사용했었어요!
둘다 괜찮아서 Given절에 세팅해둘게요!